lib/auth_browser.dart

```diff
@@ -100,6 +100,10 @@ class BrowserOAuth2Flow {
   /// already logged in) and might get asked to grant the application access
   /// (if the application hasn't been granted access before).
   ///
+  /// If [force] is `true` the user will be forced to login and asked to grant
+  /// the application access. This could be useful if the user explicitly choose
```

Comment:
I'm not sure we need to explicitly call out the existing use case, you can just use the `force` param text from a method below to keep things consistent.

Reply:
Changed using the same comment below.